src/app/fdctl/run/tiles/fd_replay.c

```diff
@@ -277,6 +272,36 @@ after_frag( void *             _ctx,
         return;
       }
 
+      // Notify for all the updated accounts
+      fd_replay_notif_msg_t * msg = NULL;
+      for( fd_funk_rec_t const * rec = fd_funk_txn_first_rec( ctx->replay->funk, fork->slot_ctx.funk_txn );
+           rec != NULL;
+           rec = fd_funk_txn_next_rec( ctx->replay->funk, rec ) ) {
+        if( !fd_funk_key_is_acc( rec->pair.key ) ) continue;
+        if( msg == NULL ) {
+          msg = fd_chunk_to_laddr( ctx->notif_out_mem, ctx->notif_out_chunk );
+          msg->type = FD_REPLAY_SAVED_TYPE;
+          msg->acct_saved.funk_xid = rec->pair.xid[0];
+          msg->acct_saved.acct_id_cnt = 0;
+        }
+        fd_memcpy( msg->acct_saved.acct_id[ msg->acct_saved.acct_id_cnt++ ].uc, rec->pair.key->uc, sizeof(fd_pubkey_t) );
+        if( msg->acct_saved.acct_id_cnt == FD_REPLAY_NOTIF_ACCT_MAX ) {
+          fd_mcache_publish( ctx->notif_out_mcache, ctx->notif_out_depth, ctx->notif_out_seq, 0UL, ctx->notif_out_chunk,
+                             sizeof(fd_replay_notif_msg_t), 0UL, 0UL, 0UL );
+          ctx->notif_out_seq   = fd_seq_inc( ctx->notif_out_seq, 1UL );
+          ctx->notif_out_chunk = fd_dcache_compact_next( ctx->notif_out_chunk, sizeof(fd_replay_notif_msg_t),
+                                                         ctx->notif_out_chunk0, ctx->notif_out_wmark );
+          msg = NULL;
+        }
+      }
+      if( msg ) {
+        fd_mcache_publish( ctx->notif_out_mcache, ctx->notif_out_depth, ctx->notif_out_seq, 0UL, ctx->notif_out_chunk,
+                           sizeof(fd_replay_notif_msg_t), 0UL, 0UL, 0UL );
+        ctx->notif_out_seq   = fd_seq_inc( ctx->notif_out_seq, 1UL );
+        ctx->notif_out_chunk = fd_dcache_compact_next( ctx->notif_out_chunk, sizeof(fd_replay_notif_msg_t),
+                                                       ctx->notif_out_chunk0, ctx->notif_out_wmark );
+      }
+
```

Comment:
We may want to send some slot metadata at some point, but it is fine for now if you do not need it. For example, slot number, blockhash. It is mostly in the bank anyways so you can decode it but just a thought.

Reply:
the slot number is in the xid